modules/install/pages/install.adoc

```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
 
 == References
 
-* Links to all relevant product docs
+* https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/8.0/html/red_hat_jboss_enterprise_application_platform_installation_methods/index[Red Hat JBoss Enterprise Application Platform Installation Methods]
+
 
 == Lab 1: Installing JBoss EAP using the Zip file
```

Comment:
add a NOTE saying that the install instructions are for RHEL 9 and that students should adjust the instructions for their operating system